packages/ember-routing/lib/system/router.js

```diff
@@ -609,7 +609,6 @@ const EmberRouter = EmberObject.extend(Evented, {
       }
 
       handler._setRouteName(routeName);
-      handler._populateQPMeta();
```

Comment:
So you've removed this to enable lazy evaluating of qp meta, but where is this lazy evaluation happening? How is this backwards compatible?